tutorials/quick_start/README.md

```diff
@@ -167,6 +167,15 @@ in the future.
         <td>30.18</td>
         <td>3.4</td>
     </tr>
+    <tr>
+        <td>Pose Estimation</td>
+        <td>yolov8n-pose</td>
+        <td>COCO</td>
```

Comment:
The performance results at the rest of the table appear with '%' after them.
Please keep this consistent (fix also in the other tabs under ultralitycs).